fault-proof/src/contract.rs

```diff
@@ -44,6 +44,9 @@ sol! {
 
     #[sol(rpc)]
     contract OPSuccinctFaultDisputeGame {
+        /// @notice Getter for the starting block number.
```

Comment:
Is this change just for us or included upstream as well? I see the point, but guess we should stay with upstream contracts.